go/oprf/groups/ecgroup/ec_test.go

```diff
@@ -208,11 +208,20 @@ func checkPointEquality(t *testing.T, curve elliptic.Curve) {
 }
 
 func ciphersuiteFromString(t *testing.T, groupName string, verifiable bool) {
-	s := ""
+	var s, ciphName string
+
 	if verifiable {
 		s = "V"
 	}
-	ciphName := fmt.Sprintf("%sOPRF-%s-HKDF-SHA512-SSWU-RO", s, strings.ReplaceAll(groupName, "-", ""))
+
+	if groupName == "P-384" {
+		ciphName = fmt.Sprintf("%sOPRF-%s-HKDF-SHA512-ICART-RO", s, strings.ReplaceAll(groupName, "-", ""))
```

Comment:
I think should still be `SSWU` rather than `ICART`, as in: https://tools.ietf.org/html/draft-irtf-cfrg-hash-to-curve-05#section-8.4

Reply:
Oh! I will change it. It is just that in the draft https://github.com/cfrg/draft-irtf-cfrg-voprf/blob/master/draft-irtf-cfrg-voprf.md#voprf-p384-hkdf-sha512-icart it was referred as that ;)